### PR TITLE
[Sensor] Addition of initial sensor values

### PR DESCRIPTION
**Pull request by pvschakradhar**
Updating initial data values of all the sensors

### CODE REVIEW

File: src/Tizen.Sensor/Tizen.Sensor/Plugins/Accelerometer.cs [hunk 1]
```diff
@@ -117,6 +117,22 @@ private static int GetCount()
             return count;
         }
 
+        private void GetInitialCount()
```
Comment (seseki17) [inline src/Tizen.Sensor/Tizen.Sensor/Plugins/Accelerometer.cs]: I think "Count" is not a proper word for this function name. is it right?

Comment (pvschakradhar) [reply src/Tizen.Sensor/Tizen.Sensor/Plugins/Accelerometer.cs]: Yes. I will replace it with Data.
"GetInitialData()", will this be fine ?

File: src/Tizen.Sensor/Tizen.Sensor/Sensor.cs [hunk 2]
```diff
@@ -47,6 +47,10 @@ public abstract class Sensor : IDisposable
         private IntPtr _sensorHandle = IntPtr.Zero;
         private IntPtr _listenerHandle = IntPtr.Zero;
 
+        /// <summary>
+        /// Read initial sensor data.
```
Comment (myroot) [inline src/Tizen.Sensor/Tizen.Sensor/Sensor.cs]: Read a sensor data synchronously.

Comment (pvschakradhar) [reply src/Tizen.Sensor/Tizen.Sensor/Sensor.cs]: comment updated, Thank you

File: src/Tizen.Sensor/Tizen.Sensor/Sensor.cs [hunk 3]
```diff
@@ -47,6 +47,10 @@ public abstract class Sensor : IDisposable
         private IntPtr _sensorHandle = IntPtr.Zero;
         private IntPtr _listenerHandle = IntPtr.Zero;
 
+        /// <summary>
+        /// Read a sensor data synchronously.
+        /// </summary>
+        protected virtual void ReadData() { }
```
Comment (myroot) [inline src/Tizen.Sensor/Tizen.Sensor/Sensor.cs]: `abstract` is more correct to enforce implementation it on base class
```suggestion
        protected abstract void ReadData() ;
```

Comment (pvschakradhar) [reply src/Tizen.Sensor/Tizen.Sensor/Sensor.cs]: yes, true that.

**Comment by bwikbs** [comment]
At this point, This patch requires ACR. And our intention is not to release ``ReadData`` API to public.
So.. plz, change ``protected`` to ``internal``

**Comment by myroot** [comment]
> At this point, This patch requires ACR. And our intention is not to release `ReadData` API to public.
> So.. plz, change `protected` to `internal`
`internal protected` is more proper
https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/keywords/protected-internal

**Comment by pvschakradhar** [comment]
> > At this point, This patch requires ACR. And our intention is not to release `ReadData` API to public.
> > So.. plz, change `protected` to `internal`
> 
> `protected internal` is more proper
> https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/keywords/protected-internal

but while opting for protected internal,  we still need to release ReadData API to public since it possesses both protected and internal behaviour

**Comment by myroot** [comment]
> > > At this point, This patch requires ACR. And our intention is not to release `ReadData` API to public.
> > > So.. plz, change `protected` to `internal`
> > 
> > 
> > `protected internal` is more proper
> > https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/keywords/protected-internal
> 
> but while opting for protected internal, we still need to release ReadData API to public since it possesses both protected and internal behaviour


No, protected internal is not public api

**Comment by myroot** [comment]
> 
> 
> > > > At this point, This patch requires ACR. And our intention is not to release `ReadData` API to public.
> > > > So.. plz, change `protected` to `internal`
> > > 
> > > 
> > > `protected internal` is more proper
> > > https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/keywords/protected-internal
> > 
> > 
> > but while opting for protected internal, we still need to release ReadData API to public since it possesses both protected and internal behaviour
> 
> No, protected internal is not public api

My fault, It should be `internal`, sorry

**Comment by bwikbs** [comment]
@pvschakradhar Thanks :smile:

**Pull request merged**